mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionDaoImpl.java

```diff
@@ -109,6 +106,13 @@ public class JobExecutionDaoImpl implements JobExecutionDao {
   public static final String TENANT_NAME = "tenantName";
   public static final String TRUE = "true";
   public static final String DB_TABLE_NAME_FIELD = "tableName";
+  public static final String SELECT_ID_WHERE_IS_DELETED_TRUE = "SELECT id FROM %s WHERE is_deleted = true and completed_date<= '%s'";
+  public static final String DELETE_QUERY = "DELETE from %s where %s IN ('%s')";
```

Comment:
As mentioned before, I am not fond of this way of writing sql...

strings in the `IN` clause need to be parameterized. In its current state, the db will need to parse every sql statement because it will appear new. This comment is applicable to the SELECT statement above as well.

Reply:
Changed both queries to be parameterized, thanks

Reply:
looks great!